libraries/procedural/src/procedural/ProceduralMaterialCache.cpp

```diff
@@ -136,12 +149,12 @@ NetworkMaterialResource::ParsedMaterials NetworkMaterialResource::parseJSONMater
  *     Set to <code>"fallthrough"</code> to fall through to the material below. <code>"hifi_pbr"</code> model only.
  * @property {number|string} scattering - The scattering, range <code>0.0</code> &ndash; <code>1.0</code>. 
  *     Set to <code>"fallthrough"</code> to fall through to the material below. <code>"hifi_pbr"</code> model only.
- * @property {string} emissiveMap - The URL of the emissive texture image. 
+ * @property {string} emissiveMap - The URL of the emissive texture image, or an entity ID.
```

Comment:
```suggestion
 * @property {string} emissiveMap - The URL of the emissive texture image, or a Material entity ID.
```
I assume? Or perhaps it's an Image entity? Similarly, below.